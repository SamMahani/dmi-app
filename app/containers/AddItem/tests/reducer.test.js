import { fromJS } from 'immutable';
import addItemReducer from '../reducer';

describe('addItemReducer', () => {
  it('returns the initial state', () => {
    expect(addItemReducer(undefined, {})).toEqual(fromJS({}));
  });
});
