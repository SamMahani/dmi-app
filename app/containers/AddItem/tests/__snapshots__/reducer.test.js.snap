// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addItemReducer should handle ADD_ITEM_SUCCESS action 1`] = `
Immutable.Map {
  "item": "",
  "success": true,
  "error": false,
}
`;

exports[`addItemReducer should properly handle addItemError action 1`] = `
Immutable.Map {
  "item": "",
  "success": false,
  "error": true,
}
`;

exports[`addItemReducer should properly handle addItemSuccess action 1`] = `
Immutable.Map {
  "item": "",
  "success": true,
  "error": false,
}
`;

exports[`addItemReducer should properly handle getItemsSuccess action 1`] = `
Immutable.Map {
  "item": "Test item",
  "success": false,
  "error": false,
}
`;

exports[`addItemReducer should return initial state when actions to not match 1`] = `
Immutable.Map {
  "item": "",
  "success": false,
  "error": false,
}
`;
